app/controllers/chargeback_rate_controller.rb

```diff
@@ -12,10 +12,10 @@ class ChargebackRateController < ApplicationController
   include Mixins::BreadcrumbsMixin
 
   BUTTON_ALLOWED_ACTIONS = {
-    'chargeback_rates_copy'   => :cb_rate_edit,
+    'chargeback_rates_copy'   => :copy,
     'chargeback_rates_delete' => :delete,
-    'chargeback_rates_edit'   => :cb_rate_edit,
-    'chargeback_rates_new'    => :cb_rate_edit
+    'chargeback_rates_edit'   => :edit,
+    'chargeback_rates_new'    => :new
   }.freeze
 
   def button
```

Comment:
I'm wondering if this could be totally removed and the buttons could have the direct links instead. It could simplify stuff there.

Reply:
@skateman please review my changes in `0479c53124b1c55f90df2003464782c43ff8d172`, encountered a bug in toolbar code where it was not sending up ids of selected checkboxes when button method is by passed.